docs/resources/cse_microservice.md

```diff
@@ -9,54 +9,87 @@ description: ""
 
 Manages a dedicated microservice resource within HuaweiCloud.
 
--> When deleting a microservice, all instances under it will also be deleted together.
+-> 1. Before creating a configuration, make sure the engine has enabled the rules shown in the appendix
+   [table](#microservice_default_engine_access_rules).
+   <br/> 2. When deleting a microservice, all instances under it will also be deleted together.
 
 ## Example Usage
 
 ### Create a microservice in an engine with RBAC authentication disabled
 
 ```hcl
-variable "engine_conn_addr" {}
+variable "microservice_engine_id" {} // Enable the EIP access
 variable "service_name" {}
 variable "app_name" {}
 
+data "huaweicloud_cse_microservice_engines" "test" {}
+
+locals {
+  fileter_engines = [for o in data.huaweicloud_cse_microservice_engines.test.engines : o if o.id == var.microservice_engine_id]
+}
+
 resource "huaweicloud_cse_microservice" "test" {
-  connect_address = var.engine_conn_addr
-  name            = var.service_name
-  version         = "1.0.0"
-  environment     = "development"
-  app_name        = var.app_name
+  auth_address    = local.fileter_engines[0].service_registry_addresses[0].public
+  connect_address = local.fileter_engines[0].service_registry_addresses[0].public
+
+  name        = var.service_name
+  version     = "1.0.0"
+  environment = "development"
+  app_name    = var.app_name
 }
 ```
 
 ### Create a microservice in an engine with RBAC authentication enabled
 
 ```hcl
-variable "engine_conn_addr" {}
+variable "microservice_engine_id" {} // Enable the EIP access
```

Comment:
This variable field is not found in this example usage.

Reply:
done.